README.md

```diff
@@ -1,2 +1,51 @@
-# eesyapi
-An R package with useful utility functions for connecting to, and processing data from, the DfE's Explore Education Statistics (EES) API.
+# <a href="https://dfe-analytical-services.github.io/eesyapi/">eesyapi</a>
+
+<!-- badges: start -->
+[![R-CMD-check](https://github.com/dfe-analytical-services/eesyapi/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/dfe-analytical-services/eesyapi/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/dfe-analytical-services/eesyapi/branch/main/graph/badge.svg)](https://app.codecov.io/gh/dfe-analytical-services/eesyapi?branch=main)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+<!-- badges: end -->
+
+An R package to support analysts in connecting to and processing data from 
+the DfE's Explore Education Statistics API endpoints.
+
+## Installation
+
+eesyapi is not currently available on CRAN. For the time being you can
+install the development version from GitHub.
+
+If you are using
+[renv](https://rstudio.github.io/renv/articles/renv.html) in your
+project (recommended):
+
+``` r
+renv::install("dfe-analytical-services/eesyapi")
+```
+
+Otherwise:
+
+``` r
+# install.packages("devtools")
+devtools::install_github("dfe-analytical-services/eesyapi")
+```
+
+## Potential errors when installing
+
+If you get `ERROR [curl: (22) The requested URL returned error: 401]`, and don't 
```

Comment:
Worth making it clear this is specifically aimed at analysts within DfE?

Reply:
Done